doc/classes/@GlobalScope.xml

```diff
@@ -370,7 +370,7 @@
 				Returns an "eased" value of [param x] based on an easing function defined with [param curve]. This easing function is based on an exponent. The [param curve] can be any floating-point number, with specific values leading to the following behaviors:
 				[codeblock lang=text]
 				- Lower than -1.0 (exclusive): Ease in-out
-				- 1.0: Linear
+				- -1.0: Linear
```

Comment:
The change makes sense to fix a typo.

However, using a codeblock as a list seems weird. The class reference can do actual bullet pointed lists:
```diff
-				[codeblock lang=text]
-				- Lower than -1.0 (exclusive): Ease in-out
-				- -1.0: Linear
-				- Between -1.0 and 0.0 (exclusive): Ease out-in
-				- 0.0: Constant
-				- Between 0.0 to 1.0 (exclusive): Ease out
-				- 1.0: Linear
-				- Greater than 1.0 (exclusive): Ease in
-				[/codeblock]
+				- Lower than [code]-1.0[/code] (exclusive): Ease in-out
+				- [code]-1.0[/code]: Linear
+				- Between [code]-1.0[/code] and [code]0.0[/code] (exclusive): Ease out-in
+				- [code]0.0[/code]: Constant
+				- Between [code]0.0[/code] and [code]1.0[/code] (exclusive): Ease out
+				- [code]1.0[/code]: Linear
+				- Greater than [code]1.0[/code] (exclusive): Ease in
```

See [JSONRPC.make_request()](https://docs.godotengine.org/en/stable/classes/class_jsonrpc.html#class-jsonrpc-method-make-request) for an example of how this will render:
https://github.com/godotengine/godot/blob/0c45ace151f25de2ca54fe7a46b6f077be32ba6f/doc/classes/JSONRPC.xml#L27-L32